src/tsconfig.json

```diff
@@ -1,5 +1,9 @@
 {
-  "include": [ "../@types", "./"],
+  "include": [
+    "../@types",
+    "../ts-defs-src",
```

Comment:
`"../ts-defs-src"`
Why does it necessary? I as remember we have a separate Gulp [step](https://github.com/DevExpress/testcafe/blob/master/Gulpfile.js#L227) for the TypeScript definitions.

Reply:
I guess it should test the compatibility between our end-user TS defs and our source-level defs.

Reply:
I've added the end user TS defs in order to reuse them in the source code and don't maintain two versions of the same TS defs.

For example  [`NodeSnapshot`](https://github.com/DevExpress/testcafe/pull/6191/files#diff-997e4f44307a245049ef83b59a672b489463b06bd4a1d571557cbb3bed03845fR1) was already defined in the end-user TS defs: 
https://github.com/DevExpress/testcafe/blob/c72b8cd2ad88f09d8194a5f48ee3d954fb2b0aae/ts-defs-src/test-api/node-snapshot.d.ts#L29 
Also the [`CompilerOptions`](https://github.com/DevExpress/testcafe/pull/6191/files#diff-0269d3bc8b9fe8b07e269b1f3928a88d59eaf0fe4f52d6a63ee3d9d7dc5f302fL15) interface was defined too: https://github.com/DevExpress/testcafe/blob/c72b8cd2ad88f09d8194a5f48ee3d954fb2b0aae/ts-defs-src/runner-api/runner-api.d.ts#L108